### ISSUE TITLE
[4.x] Existing replicator content not showing

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#7768]
### Bug description

I'm not really sure what's going on here, but existing replicator content appears partly empty:
<img width="676" alt="Screenshot 2023-03-25 at 15 07 58" src="https://user-images.githubusercontent.com/69107412/227722191-d4b5df9c-cde3-4fe6-ae1d-a95f5ff5aaa2.png">

In the Bard there should be text. And in the link blocks there should be blocks.

Here's some JS errors that come with this:
<img width="676" alt="Screenshot 2023-03-25 at 15 08 44" src="https://user-images.githubusercontent.com/69107412/227722247-d4be677a-c26c-430f-a397-199eb9ce16ae.png">

### How to reproduce

Install Peak
Upgrade it to v4
Check out the home page content for example

### Environment

```yaml
Environment
Application Name: V4
Laravel Version: 9.52.4
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
URL: v4.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 4
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: dev-master PRO

Statamic Addons
studio1902/statamic-peak-browser-appearance: 1.0
studio1902/statamic-peak-commands: 1.26
studio1902/statamic-peak-seo: 1.7
studio1902/statamic-peak-tools: 2.2
```

### Installation

Starter Kit using via CLI

### Antlers Parser

runtime (new)

**Comment by jasonvarga** [comment]
Does this happen to you now that 4.0.0-alpha.1 has been tagged? I can't reproduce.

**Comment by robdekort** [comment]
This seems to work correctly now when I do a fresh alpha install. 🎉

**Comment by robdekort** [comment]
Recreated by Jason.

**Comment by jasonvarga** [comment]
Fixed by #7827
